ui/lib/kv/addon/routes/list-directory.js

```diff
@@ -34,6 +34,9 @@ export default class KvSecretsListRoute extends Route {
           : models;
       })
       .catch((err) => {
+        if (err.httpStatus === 403) {
```

Comment:
Nit: could we move this `kv/metadata` query to its own function and use `try/catch`syntax? No need to change since I think that was previously reviewed